x-pack/heartbeat/Jenkinsfile.yml

```
@@ -13,6 +13,63 @@ when:
     tags: true                 ## for all the tags
 platform: "immutable && ubuntu-18" ## default label for all the stages
 stages:
+    Lint:
```

```suggestion
    lint:
```
Should this be capatalized?

---

`Lint` was there for other modules, so I'll say this is something we can tackle with a follow up to normalise the `Lint` stage across all the beats